src/Components/Steps/ZipcodeStep.tsx

```diff
@@ -50,27 +61,19 @@ export const ZipcodeStep = () => {
   });
 
   const currentZipcodeValue = watch('zipcode');
-  const shouldShowCountyInput = zipcodeSchema.safeParse(currentZipcodeValue).success;
+  const parsedZipCode = zipcodeSchema.safeParse(currentZipcodeValue);
```

Comment:
What do you think about using the original expression? It seems more readable and explicit in terms of why the parsed zip code is being used.

Reply:
This fixes a TS error, and allows us to use the parsed zip code in multiple places.